games.php

```diff
@@ -4,18 +4,23 @@
 
 include('./config.php');
 
-function order_servers($a, $b)
+function order_servers($servers)
 {
-    // Find the offset of the first alphabetic character in the server name
-    preg_match('~[a-z]~i', $a['name'], $a_start, PREG_OFFSET_CAPTURE);
-    preg_match('~[a-z]~i', $b['name'], $b_start, PREG_OFFSET_CAPTURE);
-
-    // Compare the alphabetic portion only, ignoring special characters at the start
-    // Servers without any alphabetic characters are pushed to the end
-    $a_comp = $a_start ? substr($a['name'], $a_start[0][1]) : "z" . $a['name'];
-    $b_comp = $b_start ? substr($b['name'], $b_start[0][1]) : "z" . $b['name'];
-
-    return strcasecmp($a_comp, $b_comp);
+	// Create an array of server names, with any non-alphanumeric characters replaced with z
+	$names = array();
+	foreach ($servers as $key => $server)
+		$names[$key] = strtolower(preg_replace("/[^\p{L}|\p{N}]/u", 'z', Normalizer::normalize($server['name'])));
+	
```

Comment:
Sorry, one more style nit: this and the two empty lines below shouldn't be indented.